deploy-if-version-bump.sh

```diff
@@ -0,0 +1,13 @@
+# deploy-if-version-bump.sh
+#!/bin/sh
```

Comment:
If you do rely on `bash`-specific features, indeed then specify `bash`. However, for portability, it is better to stay with `sh`, because `sh` is a standard more than an implementation. Specifying something else means indeed you opt-in to more specific features.